oak/server/enclave_server.h

```diff
@@ -139,61 +141,45 @@ class EnclaveServer final : public asylo::TrustedApplication {
     }
   }
 
-  // Creates a new gRPC event tag based on a number. gRPC tags are usually employed to uniquely
-  // identify events, but we do not need to keep track of them, and therefore we just create them
-  // based on arbitrary numbers.
-  // See https://grpc.io/grpc/cpp/classgrpc_1_1_completion_queue.html.
-  static void *tag(int i) { return reinterpret_cast<void *>(i); }
-
-  // Processes the next available event from the completion queue, blocking if none is available.
-  void ProcessNextEvent() {
-    void *out_tag;
-    bool ok = false;
-    completion_queue_->Next(&out_tag, &ok);
-    // TODO: Return the value of ok so that we can break the completion queue loop.
-    if (!ok) {
-      LOG(INFO) << "Could not process gRPC event from completion queue";
-    }
+  void RequestNextCall() {
+    // The stream will delete itself when it finishes.
+    auto stream = new grpc_server::GrpcStream(node_);
```

Comment:
I made this a shared_ptr - is that reasonable? I tried to make it a unique_ptr, but I think the code in the enclave was crashing but not outputting any error messages. It may be possible to do it, though.